Don't install doxytag file for now

This doxytag file makes doxygen segfault (at least 1.8.15) when other depending
packages parse the single <subgroup> element present in the file. Since there
is no obvious fix nor workaround for it, just cancel the file installation
until a better solution is found.

--- cmake/doxygen.cmake.orig	2023-01-09 13:05:22.000000000 +0100
+++ cmake/doxygen.cmake	2023-01-19 15:10:13.955183462 +0100
@@ -582,8 +582,6 @@
     # Install generated files.
     if(INSTALL_DOCUMENTATION)
       if(EXISTS ${PROJECT_BINARY_DIR}/doc/${PROJECT_NAME}.doxytag)
-        install(FILES ${PROJECT_BINARY_DIR}/doc/${PROJECT_NAME}.doxytag
-                DESTINATION ${CMAKE_INSTALL_DOCDIR}/doxygen-html)
       endif()
       install(DIRECTORY ${PROJECT_BINARY_DIR}/doc/doxygen-html
               DESTINATION ${CMAKE_INSTALL_DOCDIR})
